### ISSUE TITLE
How can i download .webm file and process without fs on node.js

### ISSUE THREAD

**Issue by okeoke85** [Jsonize/js-ffmpeg#8]
It is a question actually, I want to download a video with .webm format from google services and get frames on the fly from it, my application is a function so i cant use fs; thanks in advance.

let videoUrl = "https://a"

let output = image[]

**Comment by ntshcalleia** [comment]
Hi @okeoke85, ffmpeg accepts URLs as input, so this should work:

```js
ffmpeg.ffprobe("http://betajs.com/sample.mp4").success(function(json) {
    console.log(json);
}).error(function(error) {
    console.log(error);
});

ffmpeg.ffmpeg("http://betajs.com/sample.mp4", ["-vf", "scale=320:-2"], "output.mp4").success(function(json) {
    console.log(json);
}).error(function(error) {
    console.log(error);
});
```

[For https, however, you might need to recompile ffmpeg.](https://stackoverflow.com/questions/31514949/ffmpeg-over-https-fails)

**Comment by okeoke85** [comment]
I got the error as '  Error : {"code":103,"msg":"The input file does not exist"}'
 any thoughts @ntshcalleia 

```
        let baseVideoURL = "http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4";
        
        var ffmpeg = require('ffmpeg');

        try {
            new ffmpeg(baseVideoURL, ["-vf", "scale=320:-2"], "output.mp4").success(function(json) {
                console.log(json);
            }).error(function(error) {
                console.log(error);
            });
        } catch (err) {
            console.log(`Error : ${JSON.stringify(err)}`);
        }
```

**Comment by ntshcalleia** [comment]
Oh, I think you're trying to use this package:
https://npmjs.com/package/ffmpeg

js-ffmpeg is this one:
https://npmjs.com/package/js-ffmpeg

```js
var ffmpeg = require('js-ffmpeg')
```

**Comment by okeoke85** [comment]
Yes, i was using ffmpeg, switched to js-ffmpeg i'm using google functions so can i take output as temp data, i get the error as >  Error-1, "File does not exist" i guessing maybe this is for output file, do you have any comments, thanks.

`        let baseVideoURL = "http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4";



        try {
            var ffmpeg = require('js-ffmpeg');

            ffmpeg.ffprobe_simple(baseVideoURL).success(function (json : any) {
                console.log(json);
            }).error(function (error : any) {
                console.log(`Error-1, ${JSON.stringify(error)}`);
            });

        } catch (err) {
            console.log(`Error-2, ${JSON.stringify(err)}`);
        }`

**Comment by ntshcalleia** [comment]
Oh, I see what you mean. It worked when I tested with `ffmpeg.ffprobe` and `ffmpeg.ffmpeg`, but I just tried with `ffmpeg.ffprobe_simple` and that one is throwing an error.

I'm not sure if this is a bug or if there's a reason why it doesn't accept URLs.
